readme.md

```diff
@@ -9,11 +9,11 @@ a simple way to verify that the UNIX programs you write behave as expected.
 - The tests are located in `$REPO_DIR/bats/tests`.
 - List of the Bats dependencies used in the tests:
 
-| Bats lib     | path  | repository |
-|--------------|-------|------------|
-| bats-core    | bats/lib/bats   | <https://github.com/bats-core/bats-core.git> |
+| Bats lib     | path                              | repository                                      |
+|--------------|-----------------------------------|-------------------------------------------------|
+| bats-core    | bats/lib/bats                     | <https://github.com/bats-core/bats-core.git>    |
```

Comment:
Maybe you can try adding a colon (:) to align the text in the column
| Bats lib | path  | repository |
| :----: |  :----: |  :----: | 
| bats-core    | bats/lib/bats | <https://github.com/bats-core/bats-core.git>  |
| bats-support | bats/lib/test_helper/bats-support | <https://github.com/bats-core/bats-support.git> |

Reply:
I think "Left-aligned" fits better in this case.